lib/multiSemanticRelease.js

```diff
@@ -68,7 +68,15 @@ async function multiSemanticRelease(
 
 	// Load packages from paths.
 	const packages = await Promise.all(paths.map((path) => getPackage(path, multiContext)));
-	packages.forEach((pkg) => logger.success(`Loaded package ${pkg.name}`));
+	packages.forEach((pkg) => {
+		// Once we load all the packages we can find their cross refs
+		// Make a list of local dependencies.
+		// Map dependency names (e.g. my-awesome-dep) to their actual package objects in the packages array.
+		pkg.localDeps = pkg.deps.map((d) => packages.find((p) => d === p.name)).filter(Boolean);
```

Comment:
I might be wrong here, but I think you might end up with multiple versions of the package object, rather than one.. I can't remember the referential equality rules here 😅

Reply:
Not critical, but since we have a lodash in deps: https://github.com/dhoulb/multi-semantic-release/pull/45/commits/8ef3a6c524c3763e2913a97fc27a0ffa80dd02dc